src/components/GameDisplay.tsx

```diff
@@ -44,13 +17,13 @@ export function GameDisplay () {
   const startingUiState: UiState = {
   }
 
-  const [gameState, dispatch] = useReducer(reducer, startingState)
-  const [uiState, dispatchUI] = useReducer(uiReducer, startingUiState)
+  const [gameState, dispatch] = useReducer(gameReducer, startingState)
+  const [uiState, dispatchUi] = useReducer(uiReducer, startingUiState)
```

Comment:
This is such a nitpicky, bikesheddy comment, but two-letter acronyms can still be caps
```
GMan
gMan
UIReducer
uiReducer
UrlLink
urlLink
```